build/devices/esp32/targets/esp32_st7789/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+	"include": [
+		"$(MODDABLE)/modules/drivers/st7789/manifest.json",
+	],
+	"config": {
+		"screen": "st7789",
+		"touch": "",
+	},
+	"defines": {
+		"spi": {
+			"mosi_pin": 14,
+			"miso_pin": -1,
+			"sck_pin": 27,
+		},
+		"ili9341": {
+			"cs_pin": -1,
+			"dc_pin": 12,
+			"rst_pin": 26,
+			"backlight_pin": 0,
+			"backlight_on": 1,
+			"spi_port": "HSPI_HOST",
+			"hz": 20000000,
+			"width": 240,
+			"height": 240,
```

Comment:
The trailing commas in the JSON are invalid. The Moddable SDK accepted these as valid until [October 20](https://github.com/Moddable-OpenSource/moddable/releases/tag/OS201020) release. These should be fixed in all manifests.

Reply:
Yes, that change helped me a lot :-)